x-pack/plugins/security_solution/server/lib/detection_engine/rules/read_rules.ts

```diff
@@ -20,13 +19,13 @@ import { isAlertType, ReadRuleOptions } from './types';
 export const readRules = async ({
   alertsClient,
   id,
-  ruleId,
-}: ReadRuleOptions): Promise<SanitizedAlert | null> => {
+  ruleIds,
+}: ReadRuleOptions): Promise<RuleAlertType[] | null> => {
```

Comment:
`readRules` currently takes either a single `id` which is used on the alerts client `get` api or an array of `ruleIds` which are used on the find api. I'm wondering if there is impetus for re-working the function to also accept an array of `ids` in addition to this array of `ruleIds`?

Reply:
I'll work on refactoring to take in an array of ids as well. I wasn't sure if that would result in me touching too much stuff in this PR. But I think it'll help to have the same interface between the two parameters.

Reply:
since the bug was only related to the kql filter in the alerts client `find` api and not their `get` api I don't think it's worth also refactoring the readRules function to accept an array of `ids`. This is probably tech debt now that should be resolved in 7.10.